app/helpers/vacancies_helper.rb

```diff
@@ -72,6 +72,11 @@ def pay_scale_options
   def subject_options
     @subject_options ||= Subject.all
   end
+
+  def disable_radius?
```

Comment:
I would just check `@filters.location_category_search?` from the view, this helper method just invokes it. Also `@filters.location_category_search?` returns true or false, so no point in doing the short circuit here

Reply:
👍